frontend/views/components/Contribution.vue

```diff
@@ -154,10 +173,71 @@ export default {
         el.focus()
       }
     }
+  },
+  validations: {
+    form: {
+      contribution: {
+        [L('The contribution is required')]: required,
+        [L('The contribution already exist')]: function (x) {
```

Comment:
`The contribution is required` => `Contribution required`
`The contribution already exist` => `Contribution already exists`